central/sensor/service/pipeline/deploymentevents/pipeline.go

```diff
@@ -166,7 +166,7 @@ func (s *pipelineImpl) runGeneralPipeline(ctx context.Context, deployment *stora
 	}
 
 	var err error
-	deployment.Hash, err = hashstructure.Hash(deployment, &hashstructure.HashOptions{})
+	deployment.Hash, err = hashstructure.Hash(deployment, hashstructure.FormatV2, &hashstructure.HashOptions{})
```

Comment:
So if we move this to v2 then this is identical to the code in the deduper. Should we set the deployment hash in sensor instead so we don't end up computing it in two places? Make sure the hash is ignored by sensorhash so that it'll still default the state timestamp. Then we can remove this instance?

Reply:
I added `sensorhash` to the `hash` field but I think it wasn't necessary since it already has `hash:"ignore"` which is the default used by the library, so the field shouldn't be taken into account in the detector deduper.